test/benchmark/cli.ts

```diff
@@ -190,7 +201,13 @@ $ node test/benchmark/cli -n 20
             { withFileTypes: true }
         )
             .filter(
-                (dirEntry) => dirEntry.isFile() && dirEntry.name.endsWith('.js')
+                (dirEntry) =>
+                    dirEntry.isFile() &&
+                    dirEntry.name.endsWith('.js') &&
+                    (!opts.filter.length ||
+                        opts.filter.find(
+                            (filter) => dirEntry.name.search(filter) > -1
```

Comment:
nit: Maybe `filter` can also be a regex here, so this won't work, but I tend to prefer using `String.includes`.

Reply:
I am not supporting that right now, so good call!